live/PapapiuLab/0.1.0.ini

```diff
@@ -0,0 +1,510 @@
+# Print profiles for Papapiu printers.
+# Based on the profiles from PR: https://github.com/slic3r/slic3r-profiles/pull/13 by top-gun.
```

Comment:
This is not valid for this bundle. Remove or update this line.

Reply:
Thank u very much for your code review, we will fix it now.